<launch>

<!--
  <param name="map_description" command="$(find xacro)/xacro '$(find jimmy_drive)/cfg/map.urdf.xacro'"/>
  <node name="maptag_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher">
	  <remap from="robot_description" to="map_description"/>
  </node>

  <node pkg="jimmy_drive" type="ar_localizer.py" name="localizer">
    <param name="initial_x" value="6.6"/>
    <param name="initial_y" value="3"/>
  </node>
-->
<!--
  <node pkg="robot_localization" type="ekf_localization_node" name="global_fusion">
    <param name="two_d_mode" value="true"/>
    <param name="map_frame" value="map"/>
    <param name="odom_frame" value="odom"/>
    <param name="base_link_frame" value="base_link"/>
    <param name="world_frame" value="map"/>
    <param name="odom0" value="/odom"/>
    <param name="pose0" value="/ar_pose"/>
    <rosparam param="odom0_config">[true,  true,  false,
                                false, false, true,
                                true,  false, false,
                                false, false, true,
                                false, false, false]</rosparam>
    <rosparam param="pose0_config">[true,  true,  false,
                                false, false, true,
                                false,  false, false,
                                false, false, false,
                                false, false, false]</rosparam>
  </node>
-->

  <node pkg="move_base" type="move_base" respawn="false" name="move_base" output="screen">
    <rosparam param="controller_frequency">5</rosparam>
    <rosparam file="$(find jimmy_drive)/cfg/local_costmap_params.yaml" command="load" />
    <rosparam file="$(find jimmy_drive)/cfg/global_costmap_params.yaml" command="load" />
    <rosparam file="$(find jimmy_drive)/cfg/base_local_planner_params.yaml" command="load" />
  </node>

</launch>